src/sas/qtgui/MainWindow/GuiManager.py

```diff
@@ -382,7 +380,16 @@ def perspectiveChanged(self, new_perspective_name: str):
         # but new_perspective is of type Perspective, thus call to Perspective members are safe
         new_perspective = self.loadedPerspectives[new_perspective_name]
 
+        # Report options
         self._workspace.actionReport.setEnabled(new_perspective.supports_reports)
+
+        # Copy paste options
+        self._workspace.actionCopy.setEnabled(new_perspective.supports_copy)
+        self._workspace.actionLatex.setEnabled(new_perspective.supports_copy_latex)
```

Comment:
Aren't the specialised options redundant? If you can copy parameters, you can surely convert them into latex and excel.
Using only `perspective.supports_copy` is enough to toggle the state of all these actions, I think.

Reply:
They have significantly different content, and its very contextual. Over generalising at this level will cause issues, exactly the kind of issues that I've spent time fixing in this PR. Need to treat things that are different as different.